src/select/select.ts

```diff
@@ -259,6 +259,7 @@ export class SelectComponent implements OnInit, ControlValueAccessor {
   @Input() public textField:string = 'text';
   @Input() public childrenField:string = 'children';
   @Input() public multiple:boolean = false;
+  @Input() public formControl = new FormControl(); 
```

Comment:
Yes that's what you could already do!

Don't have an example right away but when implementing ControlValueAccessor you immediately can bind this component either by using [(ngModel)] or by using [formControl] or of course formControlName. 
I'll try and find one (or make one) for you.